### ISSUE TITLE
Results on SLidR+SEEM

### ISSUE THREAD

**Issue by NinjaCXX** [youquanl/Segment-Any-Point-Cloud#10]
Thanks for sharing. Recently, Using your code, I generated the superpixels with SEEM. Then I pre-trained the 3D backbone with SLidR with the new superpixels. However, the performance is not as good as that in paper. Specifically, I finetuned the model on 1% nuScenes dataset, the mIoU is 0.410, but the claimed result in the paper is 44.02 m_IoU. Can you give me some suggestions? Thanks very much.

**Comment by scutwi-zhy** [comment]
> Thanks for sharing. Recently, using your code, I generated the superpixels with SEEM. Then I pre-trained the 3D backbone with SLidR with the new superpixels. However, the performance is not as good as that in paper. Specifically, I finetuned the model on 1% nuScenes dataset, the mIoU is 41.0, but the claimed result in the paper is 44.02 m_IoU. Can you give me some suggestions? Thanks very much.

Sorry to bother you. Could you tell me how to pretrain the 3D backbone with SLidR with the new superpixels? Thanks very much.

**Comment by NinjaCXX** [comment]
@scutwi-zhy I pre-train the 3D backbone with the code released by SLiDR by replacing the original superpixels with the newly generated superpixels.

**Comment by scutwi-zhy** [comment]
> @scutwi-zhy I pre-train the 3D backbone with the code released by SLiDR by replacing the original superpixels with the newly generated superpixels.

Thanks for your replying！Did you make any changes in SLidR/pretrain/dataloader_nuscenes.py/map_pointcloud_to_image when you pre-train the 3D backbone？When I pre-train the 3D backbone with the newly generated superpixels, gradient explosion has occurred. Thank you and have a nice day!

**Comment by NinjaCXX** [comment]
I only changed the superpixels_type in the config file.

**Comment by scutwi-zhy** [comment]
> I only changed the superpixels_type in the config file.

Thanks very much！

**Comment by scutwi-zhy** [comment]
> Thanks for sharing. Recently, using your code, I generated the superpixels with SEEM. Then I pre-trained the 3D backbone with SLidR with the new superpixels. However, the performance is not as good as that in paper. Specifically, I finetuned the model on 1% nuScenes dataset, the mIoU is 41.0, but the claimed result in the paper is 44.02 m_IoU. Can you give me some suggestions? Thanks very much.

Sorry to bother you again. Could you share the generated superpixels with SEEM for me？I'd like to check if it's a problem with my generated superpixels or my modified code. Thanks you very much and have a nice day！ My email：wizhy@mail.scut.edu.cn

**Comment by minsikJeon** [comment]
> Thanks for sharing. Recently, using your code, I generated the superpixels with SEEM. Then I pre-trained the 3D backbone with SLidR with the new superpixels. However, the performance is not as good as that in paper. Specifically, I finetuned the model on 1% nuScenes dataset, the mIoU is 41.0, but the claimed result in the paper is 44.02 m_IoU. Can you give me some suggestions? Thanks very much.

Are there any recent updates regarding this issue? Thanks.
